skrf/vectorFitting.py

```diff
@@ -6,39 +6,19 @@
 if TYPE_CHECKING:
     from .network import Network
 
-from functools import wraps
 try:
     from . import plotting    # will perform the correct setup for matplotlib before it is called below
-    import matplotlib.pyplot as mplt
+    from .plotting import check_plotting
+    import matplotlib.pyplot as plt
```

Comment:
Why this change from `mplt` to `plt`? Can't we just leave it as is? 

As far as I know, it's only a matter of personal taste without a right or wrong. I'm pretty sure I will get errors when trying to write `mplt` in a future update of the vector fitting code. This also introduces many useless pseudo changes in the code.

Reply:
I understand this. It is not my intention to enforce `plt` or `mplt` or any other mtplotlib.pyplot alias.
The reason was that the alias appears in the `check_plotting` wrapper in plotting.py so I picked one of two.
I wonder if I can add a parameter to the wrapper to just fit the syntax of the file it is used.

Reply:
@Vinc0110 I reverted back to `mplt` in `vectorFitting.py` and find a more creative solution to accommodate different alias names with a single wrapper.

Reply:
Very good. And sorry for the trouble. Had I known there was a good reason for this change, I'd be absolutely fine with it. Anyway, your solution is even better, thanks!